rfi_file_monitor/utils/__init__.py

```diff
@@ -114,7 +114,9 @@ def get_patterns_from_string(input: str, defaults: Optional[List[str]]=None) ->
             return ['*']
     else:
         if input or input.strip():
-             return list(map(lambda x: x.strip(), input.split(','))) + defaults
+             print(defaults)
+             return list(map(lambda x: x.strip(), input.split(','))) + list(defaults)
```

Comment:
This should not be necessary, `defaults` must either `None` or `List[str]`. The cast should happen, if necessary, before the method is called instead.

Reply:
this threw an error for me... so I needed to cast, but I can cast it before the method is called.